XLS-0033d-CFTs/README.md

```diff
@@ -0,0 +1,836 @@
+
+If this operation succeeds, the corresponding **`CFTokenIssuance`** is removed and the owner’s reserve requirement is reduced by one.
+
+#### 1.3.2.1 Transaction-specific Fields
+
+| Field Name        | Required? | JSON Type | Internal Type |
+| ----------        | --------- | --------- | ------------- |
+| `TransactionType` |  ✔️        | `string`  | `UINT16`      | 
+
+Indicates the new transaction type **`CFTokenIssuanceDestroy`**. The integer value is `26` (TODO).
+
+| Field Name  | Required? | JSON Type | Internal Type |
+| ----------- | --------- | --------- | ------------- |
+| `CFTokenIssuance` |  ✔️        | `string`  | `UINT256`     | 
+
+Identifies the **`CFTokenIssuance`** object to be removed by the transaction.
+
+#### 1.3.2.2 Example **`CFTokenIssuanceDestroy`** JSON
+
+ ```json
+ {
+       "TransactionType": "CFTokenIssuanceDestroy",
+       "Fee": 10,
+       "CFTokenIssuance": "00070C4495F14B0E44F78A264E41713C64B5F89242540EE255534400000000000000"
+ }
+ ```
+ 
+### 1.3.3 The **`CFTokenIssuanceSet`** Transaction
+
+#### 1.3.3.1 CFTokenIssuanceSet
+| Field Name         | Required? | JSON Type | Internal Type |
+| ------------------ | --------- | --------- |---------------|
+| `TransactionType`  | ️ ✔        | `object`  | `UINT16`      |
+
+Indicates the new transaction type **`CFTokenIssuanceSet`**. The integer value is `28 (TODO)`.
+
+| Field Name  | Required? | JSON Type | Internal Type |
+| ----------- | --------- | --------- | ------------- |
+| `CFTokenIssuance` |  ✔️  | `string`  | `UINT256`     | 
+
+The `CFTokenIssuance` identifier.
+
+| Field Name      | Required?          | JSON Type | Internal Type |
+| --------------- | ------------------ | --------- | ------------- |
+| `Account`       | :heavy_check_mark: | `string`  | `ACCOUNTID`   | 
+
+An optional XRPL Address of an individual token holder balance to freeze/unfreeze. If omitted, this transaction will apply to all any accounts holding CFTs.
+
+| Field Name      | Required?          | JSON Type | Internal Type |
+| --------------- | ------------------ | --------- | ------------- |
+| `Flag`          | :heavy_check_mark: | `string`  | `UINT64`      | 
+
+#### 1.3.3.2 Example **`CFTokenIssuanceSet`** JSON
+
+ ```json
+ {
+       "TransactionType": "CFTokenIssuanceSet",
+       "Fee": 10,
+       "CFTokenIssuance": "00070C4495F14B0E44F78A264E41713C64B5F89242540EE255534400000000000000",
+       "Flags": 1
+ }
+ ```
+ 
+#### 1.3.3.1.1 CFTokenSet Flags
+Transactions of the `CFTokenFreeze` type support additional values in the Flags field, as follows:
+
+| Flag Name         | Flag Value | Description |
+|-------------------|------------|-------------|
+| `tfSetFreeze`     | ️`0x0001`  | If set, indicates that all CFT balances for this asset should be frozen. |
+| `tfSetUnFreeze`   | ️`0x0002`  | If set, indicates that all CFT balances for this asset should be unfrozen. |
```

Comment:
`tfSetUnFreeze` is supposed to be called `tfClearFreeze` instead, if the plan is to re-use `TrustSet` flag constants
```
constexpr std::uint32_t tfSetFreeze                        = 0x00100000;
constexpr std::uint32_t tfClearFreeze                      = 0x00200000;
```